src/main/scala/fr/polytechnique/cmap/cnam/etl/transformers/follow_up/FollowUpTransformerUtilities.scala

```diff
@@ -0,0 +1,92 @@
+package fr.polytechnique.cmap.cnam.etl.transformers.follow_up
+
+import java.sql.Timestamp
+import org.joda.time.{DateTime, DateTimeZone}
+import org.apache.spark.sql.Dataset
+import fr.polytechnique.cmap.cnam.etl.events.{Event, FollowUp}
+import fr.polytechnique.cmap.cnam.etl.transformers.follow_up.Columns.EndReasons
+
+object FollowUpTransformerUtilities {
+
+
+  case class PatientsValues(
```

Comment:
The name "PatientsValues" is not clear. I think we can call it PatientsFollowup or PatientsDates. What do you think ?